client/src/pages/ProductPage/ProductPage.tsx

```diff
@@ -1,46 +1,79 @@
 import style from './styles.module.scss';
-import img1 from '@/assets/product_p.png';
-import button_up from '@/assets/button_up.svg';
-import button_down from '@/assets/button_down.svg';
+import img1 from '@/assets/sample1.jpg';
```

Comment:
be more descriptive with names

although, if they are just testing purpose names, forget

otherwise give a proper name